packages/svelte/src/compiler/migrate/index.js

```diff
@@ -102,6 +103,13 @@ export function migrate(source) {
 			walk(parsed.instance.content, state, instance_script);
 		}
 
+		for (let labeled_to_remove of state.derived_labeled_statements) {
+			state.str.remove(
+				/** @type {number} */ (labeled_to_remove.start),
+				/** @type {number} */ (labeled_to_remove.end)
+			);
+		}
```

Comment:
Is it necessary to do this up here? What speaks against doing inside the `VariableDeclaration` visitor?

Reply:
Uh actually this was initially needed because i was converting even blocks with multiple assignments but that's no buono so i reverted that change and forgot that now i can do it in `VariableDeclaration`. Fixing now.